src/modules/raids/sagas.js

```diff
@@ -0,0 +1,23 @@
+import { call, put, take } from 'redux-saga/effects';
+
+import { FETCH_RAIDS, loadRaids } from './actions';
+import { getRaids } from './services';
+
+export function* watchLoadRaids() {
+  while (true) {
+    yield take(FETCH_RAIDS);
+    yield call(loadRaidSaga);
+  }
+}
+
+export function* loadRaidSaga() {
+  try {
+    const raids = yield call(getRaids);
+    console.log('Fetch successful!');
+    console.log(raids.data);
+    yield put(loadRaids(raids.data));
+  }
+  catch (error) {
+    console.log(error)
```

Comment:
Ah yep, good call on the error messaging. I probably wouldn't call the display straight on there though. Maybe we could call an action to change snackbarMessage there, and then in the snackbar saga, watch for changes to snackbarMessage, and make snackbarActive: true whenever there are any.

Mildly related: If we go with this setup, the dispatcher should be restricted to the React app (so `dispatch()` is only called in within containers/components). We shouldn't dispatch things within the sagas when we should really just be calling Redux store actions.